charts/tidb-cluster/templates/tidb-configmap.yaml

```diff
@@ -1,11 +1,10 @@
+{{/*
+tidb-initializer-job need a stable configmap name
+*/}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-{{- if .Values.enableConfigMapRollout }}
-  name: {{ template "cluster.name" . }}-tidb-{{ template "tidb-configmap.data-digest" . }}
-{{- else }}
   name: {{ template "cluster.name" . }}-tidb
```

Comment:
I think `pd-configmap.yaml` and `tikv-configmap.yaml` should also be modified.

Reply:
What's the purpose of this? `pd-configmap` and `tikv-configmap` are only intended to be referenced by the corresponding statefulset and there's no compatibility issue.

BTW, actually there **is** an issue about configmap rollout: the old configmap will be deleted when the new configmap created, which will cause the old `controller-revision`'s reference to the old configmap is invalid. Although this issue is not a problem now because of the careful statefulset upgrade procedure, it should (and will) be fixed for better maintainability.

Reply:
OK, it is not related to this issue. Another PR is more reasonable.